internal/pkg/runtime/engines/singularity/container_linux.go

```diff
@@ -1742,3 +1745,55 @@ func (c *container) addActionsMount(system *mount.System) error {
 
 	return system.Points.AddRemount(mount.BindsTag, containerDir, flags)
 }
+
+// addGenericMount transforms the plugin configuration into a series of
+// mount requests
+func (c *container) addGenericMount(system *mount.System) error {
+	for _, name := range c.engine.EngineConfig.GetPluginFuseMounts() {
+		var cfg struct {
+			Fuse struct {
+				DevFuseFd  int
+				MountPoint string
+			}
+		}
+
+		if err := c.engine.EngineConfig.GetPluginConfig(name, &cfg); err != nil {
+			sylog.Debugf("Getting plugin config: %+v\n", err)
```

Comment:
Should this be:

```suggestion
			sylog.Debugf("Failed getting plugin config: %+v\n", err)
```
Or something like that?